drivers/Aqara/aqara-lock/src/init.lua

```diff
@@ -107,23 +110,33 @@ local function event_lock_handler(driver, device, evt_name, evt_value)
   end
 end
 
-local function event_unlock_handler(driver, device, evt_name, evt_value)
-  local id, label
-  if evt_name == METHOD.RF447 then
-    evt_name = nil
-    id = nil
-    label = nil
-  elseif evt_name == METHOD.BLUETOOTH and evt_value == 2 then
-    evt_name = nil
-    id = nil
-    label = nil
-  elseif evt_value == 0x80020000 then -- one-time password
-    id = "OTP_STANDALONE"
-    label = nil
-  else
-    id, label = credential_utils.find_userLabel(driver, device, evt_value)
-  end
-  device:emit_event(Lock.lock.unlocked({ data = { method = evt_name, codeId = id, codeName = label } }))
+local function event_unlock_indoor_handler(driver, device, evt_name, evt_value)
```

Comment:
Can you add tests for these handlers?

Reply:
Is it ok to add a test case as a seperated pull-request?